x-pack/plugins/observability_solution/dataset_quality/common/translations.ts

```diff
@@ -137,6 +137,34 @@ export const flyoutShowAllText = i18n.translate('xpack.datasetQuality.flyoutShow
   defaultMessage: 'Show all',
 });
 
+export const flyoutImprovementText = i18n.translate(
+  'xpack.datasetQuality.flyoutDegradedFieldsSectionTitle',
+  {
+    defaultMessage: 'Degraded Fields',
+  }
+);
+
+export const flyoutImprovementTooltip = i18n.translate(
+  'xpack.datasetQuality.flyoutDegradedFieldsSectionTooltip',
+  {
+    defaultMessage:
+      'Set of degraded fields in the dataset. Please not that this list may not be exhaustive.',
```

Comment:
Looks like there's a typo here. "Please note..." instead of "Please not..." You can probably remove that part anyway and just say "This list may not be exhaustive." We could also change the tooltip a little more and say, "A partial list of degraded fields found in your dataset."  to imply that it's not exhaustive.

Reply:
Thank @mdbirnstiehl I will go with `"A partial list of degraded fields found in your dataset."` and update the tooltip in my next PR